Files/Helpers/NativeFileOperationsHelper.cs

```diff
@@ -224,6 +224,9 @@ IntPtr lpOverlapped
             [In] ref NativeOverlapped lpOverlapped,
             LPOVERLAPPED_COMPLETION_ROUTINE lpCompletionRoutine);
 
+        [DllImport("api-ms-win-core-errorhandling-l1-1-1.dll")]
```

Comment:
Not an issue but the docs for `Marshal.GetLastWin32Error()` state:
> This method exists because it is not reliable to make a direct platform invoke call to GetLastError to obtain this information. If you want to access this error code, you must call GetLastWin32Error instead of writing your own platform invoke definition for GetLastError and calling it.

So perhaps it's best to remove it before we use it improperly :)

Reply:
Didn't know it, thanks!